### DISCUSSION TITLE
Specific layout/theme/style suggestions

### DISCUSSION THREAD

**Discussion by ApolonTorq** [retypeapp/retype#57] [Ideas]
[Discussion 5](https://github.com/retypeapp/retype/discussions/5) is relevant background to these suggestions. The suggestions are to have [project level config](https://github.com/retypeapp/retype/blob/main/configuration/project.md) options for the following:
- Font size.
- Line Height.
- Maximum widths for the left, middle and right panes. In particular, it would be preferable to be able to configure the middle pane to take up more of the browser width, if available, on larger screens.
- Minimum widths for the left and right panes - to tailor when they disappear because of lack of available width.
- Minimum content widths for the left and right panes - to tailor when horizontal scroll bars kick in because of deep hierarchies or long titles on content.

I may not have worded the suggestions in a way that matches the terms used in your implementation. They are not intended as the exact settings used. Its more an allowance for a general direction of suggested layout configuration. The general direction of these suggestions is to retain whatever the opinionated stance that Retype takes with layout/design but provide some configuration to control the **compactness** or **density** of the information content that is displayed. Some content is better suited to a higher density of text and some is not - depending on the intended audience and purpose. Note that the beta API functionality is an area that I think would benefit from much higher density of textual content.

**Comment by DannyBen** [comment]
Yes. I for one am also interested in some tweaks. In particular, widths of the left and right side bars.

**Comment by geoffreymcgill** [comment]
We made some adjustments to the maximum width and resize breakpoints. Not everything in your list has been implemented, but in general the layout widths should be a little better as of Retype [`v2.0.0-preview`](https://github.com/retypeapp/retype/discussions/287#discussioncomment-2321442).
